bot/internal/bot/label.go

```diff
@@ -66,6 +66,12 @@ func (b *Bot) labels(ctx context.Context, files []github.PullRequestFile) ([]str
 		labels = append(labels, string(prSize(files)))
 	}
 
+	c := classifyChanges(b.c, files)
+	if c.Docs && !c.Code {
+		log.Println("Label: Adding no-changelog because this is a docs-only change.")
+		labels = append(labels, "no-changelog")
```

Comment:
Can you change the value to [this constant](https://github.com/gravitational/shared-workflows/blob/main/bot/internal/bot/changelog.go#L30)?

Reply:
Fixed in 9078af911d64ccf6459280851bf0fb783acc7258